include/alpaka/acc/AccGpuHipRt.hpp

```diff
@@ -78,7 +79,8 @@ namespace alpaka
             public block::shared::st::BlockSharedMemStHipBuiltIn,
             public block::sync::BlockSyncHipBuiltIn,
             public rand::RandHipRand,
-            public time::TimeHipBuiltIn
+            public time::TimeHipBuiltIn,
+            public concepts::Implements<ConceptAcc, void>
```

Comment:
Can you please replace the `void`s with their implementation here and in the other accelerator classes as well? So `AccGpuHipRt` here?